packages/story-editor/src/components/floatingMenu/elements/textAlign.js

```diff
@@ -17,21 +17,158 @@
 /**
  * External dependencies
  */
-import { Icons } from '@googleforcreators/design-system';
+import { useState, useRef, useEffect } from '@googleforcreators/react';
+import {
+  Icons,
+  ContextMenu,
+  ContextMenuComponents,
+  TOOLTIP_PLACEMENT,
+  useKeyDownEffect,
+} from '@googleforcreators/design-system';
 import { __ } from '@googleforcreators/i18n';
+import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
-import { IconButton } from './shared';
+import { useStory } from '../../../app';
+import { IconButton, useProperties } from './shared';
+
+const OFFSET_X = -8;
+const OFFSET_Y = 3;
+
+const SubMenuContainer = styled.div`
+  position: absolute;
+  top: calc(var(--height) + ${OFFSET_Y}px);
+  z-index: 9999;
+`;
+
+const StyledMenuButton = styled(ContextMenuComponents.MenuButton)`
+  flex: 0 0 60px;
+  height: 36px;
+  border: 1px solid #393d3f;
```

Comment:
Nit: Is there a similar color from the design system? 🤔

Reply:
Oh, I forgot about going back to that one. 🤦

Reply:
It's just `border.defaultNormal` 🙄. Done in 9bc94c32b2